cluster/environments/azure-multiple-clusters/aks-eastus-variables.tf

```diff
@@ -10,6 +10,10 @@ variable "gitops_east_path" {
   type = "string"
 }
 
+variable "gitops_east_url_branch" {
+  type = "string"
+}
```

Comment:
I would suggest adding a default value to keep configurable values in `.tfvars` at minimum.

Reply:
I tested that when it's not passed in, it will be set to "master". Or did you mean this variable should not appear in `.fvars` file?